docs/kpod-kill.1.md

```diff
@@ -0,0 +1,34 @@
+% kpod(1) kpod-kill- Kill one or more containers with a signal
+% Brent Baude
+# kpod-kill"1" "September 2017" "kpod"
+
+## NAME
+kpod kill - Kills one or more containers with a signal
+
+## SYNOPSIS
+**kpod kill [OPTIONS] CONTAINER [...]**
+
+## DESCRIPTION
+Kills one or more containers using a signal. The default signal is *KILL* but this can be overridden with the
```

Comment:
SIGKILL, I like docker description better.

       The main process inside each container specified will be sent SIGKILL,
        or any signal specified with option --signal.

Reply:
fixed.